### PR TITLE
Adds EZDB

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
# Document the changes in your pull request

This will conflict with https://github.com/yogstation13/Yogstation/pull/20847

This ports some python/Hitbox updates from TG as well as adds EZDB
It allows people to run a database locally for testing purposes (awesome)
Python is also now updated to 3.9.0

# Why is this good for the game?

# Testing

# Spriting

# Wiki Documentation

# Changelog

:cl:  Mothblocks, san7890, SpaceManiac, Sinsinins, ZephyrTFA
experimental: Adds EZDB
/:cl:

### CODE REVIEW

**Comment by Therandomhoboo** [comment]
I thought this was a new atmos gas for a second

File: .github/guides/EZDB.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+**ezdb** is a tool for quickly setting up an isolated development database. It will manage downloading MariaDB, creating the database, setting it up, and updating it when the code evolves. It is not recommended for use in production servers, but is perfect for quick development.
+
+To run ezdb, go to `tools/ezdb`, and double-click on ezdb.bat. This will set up the database on port 1338, but you can configure this with `--port`. When it is done, you should be able to launch yogstation as normal and have database access. This runs on the same Python bootstrapper as things like the map merge tool, which can sometimes be flaky.
+
+If you wish to delete the ezdb database, delete the `db` folder as well as `config/ezdb.txt`.
+
+To update ezdb, run the script again. This will both look for any updates in the database changelog, as well as update your schema revision.
+
+Contact Mothblocks if you face any issues in this process.
```
Comment (ToasterBiome) [inline .github/guides/EZDB.md:12]: As nice as Mothblocks is, I don’t think we should be bothering a tg maintainer if this breaks.

Comment (JohnFulpWillard) [reply .github/guides/EZDB.md:12]: they made it so idk who else we would bother if it breaks.

Comment (ToasterBiome) [reply .github/guides/EZDB.md:12]: you

**Comment by ToasterBiome** [comment]
you

File: .github/guides/EZDB.md (hunk 1 above)
Comment (JohnFulpWillard) [reply .github/guides/EZDB.md:12]: i would just contact mothblocks in those cases fr

**Review by ToasterBiome — APPROVED** [review]

**Comment by ynot01** [comment]
> you

image

**Pull request merged**